src/components/Modal/Modal.js

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import Button from '../Button/Button';
+import ModalContainer from './ModalContainer';
+
+const Modal = ({ content, onClose, propsFunction, isCancel }) => {
+  const handleClose = () => {
+    onClose();
+  };
```

Comment:
handleClose라는 함수가 onClose함수만 실행됬는거면 이미 onClose함수로 받고 있기 때문에 다시 함수를 만들 필요는 없을 것 같습니다.
아래에서 사용할 때 
```suggestion
onClick={onClose}
```
로 하셔도 될 것 같아요!